From 3572398c8acb1145568d1d040308433bd370433b Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 2 Nov 2023 17:40:01 +0100
Subject: [PATCH 0025/1695] plat-stm32mp1: default disable
 CFG_STM32MP1_SCMI_SHM_SYSRAM

Default disable CFG_STM32MP1_SCMI_SHM_SYSRAM to sync with mainline
Linux kernel that uses OP-TEE shared memory for SCMI communication
on STM32MP13 platform instead a piece of SRAM [1].

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 81f240895b252cbb390c18be74974590c376b3e0 plat-stm32mp1: default disable CFG_STM32MP1_SCMI_SHM_SYSRAM]
Change-Id: Ic887ada1826f302883a47012925acd24694475d5
Link: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=f0f0682c384d81bf25e6f8b23971fb8f69122f72
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from b205ee2c02ee874186c57840d0ce91ba587bda93]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index c488c5887..3f981361e 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -118,7 +118,6 @@ $(call force,CFG_TEE_CORE_NB_CORE,1)
 $(call force,CFG_WITH_NSEC_GPIOS,n)
 CFG_EXTERNAL_DT ?= n
 CFG_STM32MP_OPP_COUNT ?= 2
-CFG_STM32MP1_SCMI_SHM_SYSRAM ?= y
 CFG_WITH_PAGER ?= n
 endif # CFG_STM32MP13
 
-- 
2.47.2

